geode-wan/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ReplicateRegionCommandDUnitTest.java

```diff
@@ -220,6 +210,90 @@ public void testUnsuccessfulCancelReplicateRegionCommandInvocation_WithPartition
     testUnsuccessfulCancelReplicateRegionCommandInvocation(true, true);
   }
 
+  @Test
+  public void testUnsuccessfulExecutionDueToSenderWentDown() throws Exception {
+    testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(true);
+  }
+
+  @Test
+  public void testUnsuccessfulExecutionDueToReceiverWentDown() throws Exception {
+    testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(false);
+  }
+
+  public void testUnsuccessfulExecutionDueToSenderOrReceiverWentDown(boolean stopSender)
```

Comment:
For clarity, I would rename the parameter to something like: `stopSenderElseStopReceiver`